spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -33,16 +34,10 @@
  */
 @ConfigurationProperties("spring.cloud.stream")
 @JsonInclude(Include.NON_DEFAULT)
-public class ChannelBindingProperties {
-
-	public static final String DESTINATION = "destination";
-
-	public static final String PARTITION_COUNT = "partitionCount";
-
-	public static final String PARTITIONED = "partitioned";
+public class ChannelBindingServiceProperties {
```

Comment:
Renamed so that it doesn't overlap with `BindingProperties`. Also, it reflects better the relationship with `ChannelBindingService` (which needs to refactored and renamed, just not in this PR)